packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -102,13 +151,11 @@ export default class Dropdown extends React.Component<DropdownProps> {
     addEventListeners() {
         document.addEventListener("mouseup", this.handleInteract);
         document.addEventListener("touchend", this.handleInteract);
-        document.addEventListener("keyup", this.handleKeyup);
```

Comment:
This was originally for closing on the menu on pressing escape, now there's a general keyboard listener that handles this and navigation.